projects/graph/graph.py

```diff
@@ -46,7 +46,7 @@ def bft(self, starting_vertex):
         # while q isn't empty
         while queue.size():
                 # dequeue
-            node = queue.dequeue()
+            nyea
```

Comment:
This line will cause a syntax error.

Reply:
Sorry I missed it. I fixed it. Must have mistyped and didn't see where I was